attestor/.golangci.yml

```diff
@@ -0,0 +1,155 @@
+# Copyright 2022 Security Scorecard Authors
```

Comment:
Do we need a separate .golangci config here? Can't we re-use the one we have in the root directory?

Reply:
Removing. Was having issues with golangci not running properly, but I think those only happen on my workspace